### DISCUSSION TITLE
AudioGridder stuck in a continuous restart with Acoustica Audio plugins

### DISCUSSION THREAD

**Discussion by MrPisquan** [apohl79/audiogridder#276] [Q&A] [answered]
Hi,

thank you for this incredible software, it is changing my life!

Today I am experiencing a problem never happened to me before. When I load some Acustica Audio plugins the server begins to restart in an infinite loop.

This happens both when I recall the plugins as a preset within Cubase 11 and loading a saved project, and it never happens when I work at a new project starting from scratch.

Thank you for the support!

[AudioGridderPlugin_2020-12-19_16-15-23.zip](https://github.com/apohl79/audiogridder/files/5719782/AudioGridderPlugin_2020-12-19_16-15-23.zip)

**Comment by MrPisquan** [comment, marked as answer]
Problem solved. I uninstalled and reinstalled the Acustica Audio plugins and the problem has appeared no more. So it wasn't about Audio Gridder.
Thank you again! :-)

**Comment by apohl79** [comment]
Thanks for reporting the problem and the solution. Moved the issue into the discussions area.

**Reply by MrPisquan** [reply]
Thank you Andreas!
